Release: can't push to master directly

When tagging, gha gives the error : 
```
remote: error: GH006: Protected branch update failed for refs/heads/main.        
remote: error: At least 1 approving review is required by reviewers with write access.        
To https://github.com/InseeFrLab/sugoi-api.git
 ! [remote rejected] main -> main (protected branch hook declined)
error: failed to push some refs to 'https://github.com/InseeFrLab/sugoi-api.git'
Error: Process completed with exit code 1.
```

I don't know the best way to manage this, either unprotect the main branch, or don't update the snapshot version directly (maybe create a PR to do it). 

@InseeFrLab/dot What do you think ?

--- a/.github/workflows/tag.yml
+++ b/.github/workflows/tag.yml
@@ -97,8 +97,10 @@ jobs:
           git config --global user.email "dg75-dsi-domaine-outils-transverses@insee.fr"
           git checkout main
           old_version=${{ needs.build.outputs.version }}
-          new_version=$(echo $old_version | cut -d. -f1).$(($(echo $old_version | cut -d. -f2) + 1)).0-SNAPSHOT 
-          mvn -B versions:set -DnewVersion=$new_version -DgenerateBackupPoms=false
+          new_version=$(echo $old_version | cut -d. -f1).$(($(echo $old_version | cut -d. -f2) + 1)).0
+          git checkout -b prepare-version-$new_version
+          mvn -B versions:set -DnewVersion=$new_version-SNAPSHOT -DgenerateBackupPoms=false
           git add .
-          git commit -m"Prepare new version"
-          git push
+          git commit -sm"Prepare new version"
+          git push origin prepare-version-$new_version
+          gh pr create --title "Prepare new version" --body "Prepare new version"

